### PR TITLE
Fix tests part 1

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -9,8 +9,8 @@
         "**/node_modules": true
     },
     "editor.codeActionsOnSave": {
-        "source.fixAll": true,
-        "source.organizeImports": true
+        "source.fixAll": "explicit",
```
Comment (StephenWeatherford) [inline .vscode/settings.json:12]: vscode keeps wanting to make these changes...

File: package.json [hunk 2]
```diff
@@ -657,7 +657,6 @@
         "typescript": "^4.5.5",
         "vsce": "^1.85.0",
         "vscode-azureextensiondev": "^0.5.0",
-        "vscode-test": "^1.5.2",
```
Comment (StephenWeatherford) [inline package.json:660]: vscode-test is the old name of @vscode/test-electron

**Comment by StephenWeatherford** [comment]
Now tests failing with:
 "Error: There are problems with this document's schema impacting one or more items in the document. Please report this issue to schema owner. (arm-template (schema)) [2,14-2,95]",

Something wrong with the schema zip file probably.

**Review by AbhitejJohn — APPROVED** [review]

**Comment by StephenWeatherford** [comment]
Now getting:

"Error: There are problems with this document's schema impacting one or more items in the document. Please report this issue to schema owner. (arm-template (schema)) [3,16-3,97]",

**Pull request merged**